app/models/user.rb

```diff
@@ -36,4 +36,16 @@ class User < ApplicationRecord
   has_many :classrooms
   has_many :student_classrooms
   has_many :attending_classrooms, through: :student_classrooms, source: :classroom
+
+  # Returns the AR instances of students for a specific teacher
```

Comment:
ruby is an easy to read language so what a method does should be clear by its name
hence it's generally frowned upon to add explaining comments
please remove them